autoload/SpaceVim/layers/lang/csharp.vim

```diff
@@ -0,0 +1,109 @@
+"=============================================================================
+" csharp.vim --- SpaceVim lang#csharp layer
+" Copyright (c) 2016-2017 VyronLee & Contributors
```

Comment:
I am sorry, as we are using `GPLv3` License, so the copyright should be 

```
" Copyright (c) 2016-2017 Wang Shidong & Contributors
```

you can use your info after `Author:`.

Reply:
It's my fault.